Circuit/Circuit.csproj.old

```diff
@@ -0,0 +1,134 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Remove this .old file?

Reply:
I've made this changes, to avoid using P/Invoke Windows-specific implementations in Audio library. Parameter naming might be reverted if you like. I've changed that for API consistency, and to follow C# naming conventions (using the scout rule) ;)